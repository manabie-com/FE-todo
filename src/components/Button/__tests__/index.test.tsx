import * as React from 'react';
import { render } from '@testing-library/react';

import { Button } from '../index';

describe('<Button />', () => {
  it('should match snapshot', () => {
    const button = render(<Button />);
    expect(button.container.firstChild).toMatchSnapshot();
  });

  it('should match primary snapshot', () => {
    const button = render(<Button primary />);
    expect(button.container.firstChild).toMatchSnapshot();
  });
});
