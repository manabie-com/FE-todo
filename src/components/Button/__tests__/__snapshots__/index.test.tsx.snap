// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should match primary snapshot 1`] = `
.c0 {
  outline: none;
  cursor: pointer;
  border: none;
  font-weight: 700;
  padding: 8px 16px;
  border-radius: 6px;
  background-color: #2185d0;
  color: #fff;
}

.c0:hover,
.c0:focus {
  background-color: #1678c2;
}

.c0:active {
  background-color: #1a69a4;
}

<button
  class="c0"
/>
`;

exports[`<Button /> should match snapshot 1`] = `
.c0 {
  outline: none;
  cursor: pointer;
  border: none;
  font-weight: 700;
  padding: 8px 16px;
  border-radius: 6px;
  background-color: #e0e1e2;
  color: rgba(0,0,0,0.6);
}

.c0:hover,
.c0:focus {
  background-color: #cacbcd;
  color: rgba(0,0,0,0.8);
}

.c0:active {
  background-color: #babbbc;
  color: rgba(0,0,0,0.9);
}

<button
  class="c0"
/>
`;
